Update plugin description in code issue #33 :

oussamaLaribi opened the pull request:
c -Add homepage URL
 -Fix organization: **we should also change the organization in Administration → Organization details → Edit organization key to fixing build failure org.sonar.api.utils.MessageException: Project not found.**
 -Fix organization URL
 -Add minimal JRE Specification Version

dedece35 commented:
Hi @oussamaLaribi ,

please correct the failed checks phase.
thank you.

oussamaLaribi commented:
> Hi @oussamaLaribi ,
> 
> please correct the failed checks phase. thank you.
Hi @dedece35, 
I don't know why the sonar failed maybe I don't have permission from the new organization. Thanks.

dedece35 reviewed (commented):
why do you have to need to do this modification ?
I corrected the problem of external PR checks, yesterday .... and it's ok now

for me, non need to modify this github action

oussamaLaribi commented:
> why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> 
> for me, non need to modify this github action

Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33

dedece35 commented:
> > why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> > for me, non need to modify this github action
> 
> Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33

Hi @oussamaLaribi, ok. sorry, my bad, I saw that it's a revert and not an update :p after writing my comment :(

oussamaLaribi commented on java-plugin/pom.xml:
```diff
@@ -91,6 +91,7 @@
                     <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
                     <!-- <requirePlugins>java:${sonarjava.version}</requirePlugins> -->
                     <basePlugin>java</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
No, we don't have a problem. It's just a lack of information
Minimal JRE Specification Version: empty
When we build the plugin, mentioned in  #33

dedece35 replied:
ok

oussamaLaribi commented on php-plugin/pom.xml:
```diff
@@ -56,6 +56,7 @@
                     <sonarLintSupported>true</sonarLintSupported>
                     <sonarQubeMinVersion>${sonarqube.version}</sonarQubeMinVersion>
                     <basePlugin>php</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
same as the java part also

dedece35 replied:
ok

oussamaLaribi commented on python-plugin/pom.xml:
```diff
@@ -64,6 +64,7 @@
                     <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
                     <!-- <requirePlugins>python:${sonar.python.version}</requirePlugins> -->
                     <basePlugin>python</basePlugin>
+                    <jreMinVersion>${java.version}</jreMinVersion>
```
Same as the java part also

dedece35 replied:
ok

oussamaLaribi commented:
> > > why do you have to need to do this modification ? I corrected the problem of external PR checks, yesterday .... and it's ok now
> > > for me, non need to modify this github action
> > 
> > 
> > Hi @dedece35, yes I saw it, its ok for me too thanks, I have reverted the changes the other modification is related to issue #33
> 
> Hi @oussamaLaribi, ok. sorry, my bad, I saw that it's a revert and not an update :p after writing my comment :(

Alright, thanks anyway.

dedece35 commented:
Hi @oussamaLaribi,
PR approved ! 
you can merge with squash option and small comment, please.
